qmlti/lib.php

```diff
@@ -181,7 +187,13 @@ function init_db($db) {
 CREATE TABLE [dbo].[{$resource_link_table_name}] (
   [consumer_key] VARCHAR(50) NOT NULL DEFAULT '',
   [context_id] VARCHAR(100) NOT NULL,
+  [lti_context_id] VARCHAR(255),
+  [lti_resource_id] VARCHAR(255),
+  [title] VARCHAR(255),
   [settings] TEXT,
+  [primary_consumer_key] VARCHAR(255),
```

Comment:
Primary_consumer_key is a holdover from the previous build of the LTI (the last schema produced by the 3rd party), and is used in most of the data accessor functions. It's mostly used for accessing 'shared' keys - for example, sharing the instructor key to the user to ensure that the participants are using the right resource.

This change is simply to ensure that the correct schema is generated if needed - as this index wasn't generated in the code before, and was raising issues for us :).